src/class/object_genome_variants.php

```diff
@@ -173,6 +173,12 @@ function __construct ()
                         'chromosome' => array(
                                     'view' => array('Chr', 50),
                                     'db'   => array('vog.chromosome', 'ASC', true)),
+                        'position_g_start' => array(
+                                    'view' => false,
+                                    'db'   => array('position_g_start', 'ASC', 'INT')),
+                        'position_g_end' => array(
+                                     'view' => false,
+                                    'db'   => array('position_g_end', 'ASC', 'INT')),
```

Comment:
If you add `vog` to the column names, LOVD will know it's an INT column and the last two array elements (`'ASC', 'INT'`) won't be needed ;)

(the `ASC` is only needed if you want to make it sortable which won't work here since the column in hidden, but something needs to be specified if you need that 3rd element; so if you won't need the last anymore, you can drop the `ASC` as well)